packages/battery/example/android/app/src/main/java/io/flutter/plugins/batteryexample/EmbedderV2Activity.java

```diff
@@ -0,0 +1,12 @@
+package io.flutter.plugins.batteryexample;
```

Comment:
I would suggest putting this into the `dev.flutter.plugins.batteryexample` namespace and making the class name be `MainActivity`. This will line up with what is happening in the other migration PRs:

https://github.com/flutter/plugins/pull/2155
https://github.com/flutter/plugins/pull/2156
https://github.com/flutter/plugins/pull/2157

Reply:
Note that this Activity is here just as a place holder until we figure out the canonical structure of an example app that supports both embedders.

Regarding the `dev.flutter` namespace, I think we should reconsider whether we want to split plugins into 2 namespaces. Part of the motivation for putting up this PR as-is is to have an example to what it looks like if we don't split the code into 2 namespaces.